projects/aca-content/src/lib/components/search/search-input/search-input.component.ts

```diff
@@ -86,12 +86,14 @@ export class SearchInputComponent implements OnInit, OnDestroy {
     private router: Router,
     private store: Store<AppStore>,
     private appHookService: AppHookService,
+    private appService: AppService,
     public searchInputService: SearchNavigationService
   ) {
     this.searchOnChange = this.config.get<boolean>('search.aca:triggeredOnChange', true);
   }
 
   ngOnInit() {
+    this.appService.appNavNarMode$.next('collapsed');
```

Comment:
missing unit tests for all new lines in that file

Reply:
Added the unit test for this file

Reply:
You added only one test but here are 3 new lines which you added